### DISCUSSION TITLE
Welcome to donorconnect Discussions!

### DISCUSSION THREAD

**Discussion by prajapatihet** [prajapatihet/donorconnect#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sonu-Hansda** [comment]
I am eager to contribute to this project and leverage my proficiency in Flutter. Upon reviewing the issue list, I notice that most of them have been assigned to team members. Could you kindly assist me in identifying a suitable issue that aligns with my skills, so that I can commence contributing to the project?

**Comment by prajapatihet** [comment]
If there any feature or module is missing in project or you think the required feature is necessary or required then raise new issue and start solving it as project focus on Donation which include blood, organ, plasma, platelets etc which is legel and authorized to donate
there is one feature i think is missing like regular user who participate in donation events and donate, that user get some amount of  points or coins and i future user avail that coin for purchasing coupons like medical or related to pharma

**Comment by prajapatihet** [comment]
You can even enhance the existing screen or feature if you can
